hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileStatus.java

```diff
@@ -52,7 +58,16 @@ public boolean isEmptyDirectory() {
 
   @Override
   public String getOwner() {
```

Comment:
If we implement the above comments, then we can completely remove the override of `getOwner` here.